rag/llm/chat_model.py

```diff
@@ -68,7 +68,6 @@ def chat_streamly(self, system, history, gen_conf):
                     resp.choices[0].delta.content = ""
                 ans += resp.choices[0].delta.content
 
-                total_tokens += 1
```

Comment:
This is just for case `if not hasattr(resp, "usage") or not resp.usage:`

Reply:
But I don't understand why we need to do total_tokens += 1 every time? Indeed, it only works when if not hasattr(resp, "usage") or not resp.usage is satisfied, so why not put this statement inside the statement? 
Anyway, if total_tokens += 1 needs to be executed, then there is no problem with the current code. 
But I don't understand why we need to do this? Could you please explain it to me? Many thanks.

Reply:
Reasonable.